box.json

```diff
@@ -0,0 +1,16 @@
+{
+    "chmod": "0755",
+    "directories": [
```

Comment:
`src` and `vendor` should be unnecessary here, but it requires to make sure there is a `composer.lock` and that the dependencies have been installed (Box can remove the dev deps so a plain `composer install` is enough)

Reply:
@theofidry thanks for the hint, and sorry for the late reply. I noticed that our phar was quite big (18.5 MB) because of dev dependencies so I have been playing a bit with the `box.json`. It seems that the removal of dev dependencies does not occur when we specify the `vendor` directory in the `directories` key of `box.json`, is that correct?

Unfortunately, it seems that if I remove `src` and `vendor` from the `directories` list, they are excluded from the phar altogether (except for `vendor` which only contains `autoload.php` and the `composer` directory). If I omit the `directories` key completely, the `src` and `vendor` directories are included correctly but the `config` directory is missing. This seems to be according the documentation, but doesn't really work for us.

Is there any way to include the `config` directory but still let Box remove our dev dependencies? Specifying it using `directories-bin` seems to work, but that feels a bit like abusing the option.

Reply:
>It seems that the removal of dev dependencies does not occur when we specify the vendor directory in the directories key of box.json, is that correct?

Yes, a typical case justifying that behaviour is one that may need to actually include a dev-only directory from vendor to the PHAR.

I think `directories-bin` is correct: it doesn't need to be processed per se. The issue is if you start to use `directories` it's the sign for box that you wanna manage your dependencies more manually.